force-app/emails/lwc/emailDeliveryReport/emailDeliveryReport.html

```diff
@@ -0,0 +1,23 @@
+<template>
+    <lightning-quick-action-panel header="Email Delivery Report">
+        <h3 class="slds-text-heading_small slds-m-bottom_medium">
+            The following emails could not be sent.
```

Comment:
Is it overkill to use labels instead of text at this point? We'll need to do it eventually.